src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryUnary.cs

```diff
@@ -413,6 +413,28 @@ internal static partial class Library
                     checkRuntimeValues: DeferRuntimeValueChecking,
                     returnBehavior: ReturnBehavior.ReturnBlankIfAnyArgIsBlank,
                     targetFunction: TimeParse)
+            },
+            {
+                UnaryOpKind.TextToBlob,
+                StandardErrorHandling<StringValue>(
+                    functionName: null, // internal function, no user-facing name
+                    expandArguments: NoArgExpansion,
+                    replaceBlankValues: DoNotReplaceBlank,
+                    checkRuntimeTypes: ExactValueTypeOrBlank<StringValue>,
+                    checkRuntimeValues: DeferRuntimeValueChecking,
+                    returnBehavior: ReturnBehavior.ReturnBlankIfAnyArgIsBlank,
+                    targetFunction: TextToBlob)
+            },
+            {
+                UnaryOpKind.BlobToText,
+                StandardErrorHandling<BlobValue>(
+                    functionName: null, // internal function, no user-facing name
+                    expandArguments: NoArgExpansion,
+                    replaceBlankValues: DoNotReplaceBlank,
+                    checkRuntimeTypes: ExactValueTypeOrBlank<BlobValue>,
+                    checkRuntimeValues: DeferRuntimeValueChecking,
+                    returnBehavior: ReturnBehavior.ReturnBlankIfAnyArgIsBlank,
+                    targetFunction: BlobToText)
             },
```

Comment:
If we don't need a URI-based blob, can we remove those conversions?

Reply:
BlobToText yes
Not TextToBlob